Tests/RevenueCatUITests/CustomerCenter/ManageSubscriptionsViewModelTests.swift

```diff
@@ -316,17 +320,89 @@ class ManageSubscriptionsViewModelTests: TestCase {
     }
 
     func testLoadScreenFailure() async {
+        let mockPurchases = MockManageSubscriptionsPurchases(customerInfoError: error)
         let viewModel = ManageSubscriptionsViewModel(screen: ManageSubscriptionsViewModelTests.screen,
-                                                     purchasesProvider: MockManageSubscriptionsPurchases(
-            customerInfoError: error
-        ))
+                                                     purchasesProvider: mockPurchases,
+                                                     promotionalOfferViewModel: MockPromotionalOfferViewModel())
 
         await viewModel.loadScreen()
 
         expect(viewModel.subscriptionInformation).to(beNil())
         expect(viewModel.state) == .error(error)
     }
 
+    func testLoadsPromotionalOffer() async throws {
```

Comment:
Heads up that `PromotionalOfferViewModelTests` was taking me longer to write than expected since I was trying to reuse some of the fixtures in this test class, so it's coming in a future PR. I don't want to delay the alpha release due to tests